mods/ra/rules/structures.yaml

```diff
@@ -1352,7 +1352,6 @@ AFLD:
 		LongDesc: Reveals an area of the map\nand cloaked enemy units.
 		SelectTargetSpeechNotification: SelectTarget
 		EndChargeSpeechNotification: SpyPlaneReady
-		CameraActor: camera.spyplane
```

Comment:
camera.spyplane has `DetectsCloaked`trait. Simply replacing that with CameraRange would drop it.

Reply:
Note to future reviewers: This is already fixed in the [3rd commit](https://github.com/OpenRA/OpenRA/pull/13858/commits/b9018cd7b884b7c46f51b2945067fda72f042de5) (though it's shown as 2nd commit by github).